BackofficeBundle/Resources/public/coffee/InternalLinkFormView.coffee

```diff
@@ -0,0 +1,58 @@
+InternalLinkFormView = OrchestraView.extend(
```

Comment:
can use namespace, on the syntax with class for this backbone view and add js comment for methods view